vcpkg-configuration.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can this go into the existing vcpkg.json?

Reply:
AFAIK it can't and you have to add vcpkg-configuration.json to use overlays (https://learn.microsoft.com/en-us/vcpkg/concepts/manifest-mode#configuration-file)